examples/common/java/direct/worker/Dockerfile

```diff
@@ -1,3 +1,16 @@
+#
```

Comment:
No copyright

Reply:
I see that Dan has removed copyright line from some licence headers.  I'm not aware of why that it, and would normally expect to see them there.

Reply:
@TomBarnes The Java files were not contributed by Intel, so I did not add an Intel copyright.